src/main/java/frc/robot/subsystems/drivetrain/commands/DriveForward.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.OI;
+import frc.robot.Robot;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.valuetuner.WebConstant;
+import org.techfire225.webapp.FireLog;
+
+public class DriveForward extends CommandBase {
+
+    private SwerveDrive swerveDrive;
+    private WebConstant target = new WebConstant("targetSpeed", 0);
```

Comment:
```suggestion
    private WebConstant targetSpeed = new WebConstant("targetSpeed", 0);
```

Reply:
I don't think we need the word speed because the command is drive forward

Reply:
I only suggested it because the name of the WebConstant is targetSpeed

Reply:
We can change the name of the Webconstant